packages/venia-ui/lib/components/Image/image.js

```diff
@@ -108,7 +108,7 @@ const Image = props => {
                 alt={alt}
                 classes={classes}
                 displayPlaceholder={displayPlaceholder}
-                height={talonResourceHeight}
+                height={height}
```

Comment:
Not setting the calculate talon height on the image because this will override the `srcset` on the `img` tag. `srcset` works best with width. By providing height we are making the container have a fixed height and because of that, the image stretches.

Before:
![image](https://user-images.githubusercontent.com/35203638/83042578-6badf800-a007-11ea-8779-dc427a2930b7.png)

After:
![image](https://user-images.githubusercontent.com/35203638/83042474-50db8380-a007-11ea-8857-af84e37693d0.png)